### ISSUE TITLE
Plot and display series if SA fail (and additionnal explanations)

### ISSUE THREAD

**Issue by TanguyBarthelemy** [jdemetra/jdplus-main#283]
(Issue related to jdemetra/jdemetra-core#1129)

If the seasonnal adjustment fails, I can't neither plot nor see my values.

For example here, I have a dataset with 11 values so the seasonal adjustment will obviously fail:

![image](https://github.com/jdemetra/jdplus-main/assets/112711017/818fc2b7-9885-4627-8edc-b11553b95cf4)

So the **Status** is **invalid** (obviously), but as you can see there is no message, warning or error in the **Main results** panel:

![image](https://github.com/jdemetra/jdplus-main/assets/112711017/40f9b633-b7cc-4907-b8dc-66bc15cc8379)

In addition to that, the Input/Series and Input/Specifications panels are also empty:

![image](https://github.com/jdemetra/jdplus-main/assets/112711017/4a04a661-10ca-4cab-8814-6d4eb20c66d9)

I would like to see my data (grid and plot) and to know why the seasonal adjustment failed.

**Comment by a-s-russo** [comment]
I agree with @TanguyBarthelemy about the importance of this issue. In general, regardless of the type of error, it is important to both:

- clearly identify that an error has been encountered (e.g. with a pop-up window, a message in the status bar, or something similar); and
- explain why the error was encountered (if known).

In some other testing we have been conducting, we have experienced a similar issue to Tanguy. Certain series and combinations of parameter settings will yield a blank document when the seasonal adjustment processing fails, which is not helpful in identifying the cause of our issue.
